src/infrastructure/getBlock.js

```diff
@@ -33,27 +33,7 @@ class sdkBlock {
     return format.formatBlock(blockInfo)
   }
 
-  static getBlockFullTransactionsList = async (blockHeight, transactionId) => {
-    let txList = await this.getTransactionsByBlockHeight(blockHeight, transactionId)
-    if (txList.length > 0) {
-      transactionId = txList[txList.length - 1].transactionId
-      const page = await this.getBlockFullTransactionsList(blockHeight, transactionId)
-      txList = [...txList, ...page]
-    }
-    return txList
-  }
-
-  static getBlockLatestTransactionsList = async (blockHeight, transactionId) => {
-    let txList = await this.getTransactionsByBlockHeight(blockHeight, transactionId)
-    if (txList.length > 0) {
-      transactionId = txList[txList.length - 1].transactionId
-      const page = await this.getTransactionsByBlockHeight(blockHeight, transactionId)
-      txList = [...txList, ...page]
-    }
-    return format.formatBlocktransactions(txList)
-  }
-
-  static getBlockTransactionsFromId = async (blockHeight, transactionId) => {
+  static getBlockTransactionsFromIdSmall = async (blockHeight, transactionId) => {
```

Comment:
the name `Small` doesn't sound mean anything.
maybe just change the name `getBlockTransactions`